Makefile

```diff
@@ -295,24 +285,3 @@ docker-gen:
 .PHONY: remove-bazel-output
 remove-bazel-output:
 	bazelisk clean --expunge
-
-.PHONY: disable-expired-trial-projects
-disable-expired-trial-projects:
-	bazelisk ${BZLFLAGS} run ${BUILD_FLAGS} //hack/disable-expired-trial-projects:disable-expired-trial-projects -- disable \
-		--cert=${WEB_GATEWAY_CERT_PATH} \
-		--service-token=${SERVICE_TOKEN_PATH} \
-		--web-gateway=${WEB_GATEWAY_ADDR}:443 \
-		--no-profile \
-		--no-gcp-trace-enabled
-
-.PHONY: delete-user-data
-delete-user-data:
-	bazelisk ${BZLFLAGS} run ${BUILD_FLAGS} //hack/delete-user-data:delete-user-data -- delete \
-		--mysql-host=${MYSQL_HOST} \
-		--mysql-port=${MYSQL_PORT} \
-		--mysql-user=${MYSQL_USER} \
-		--mysql-pass=${MYSQL_PASS} \
-		--mysql-db-name=${MYSQL_DB_NAME} \
-		--target-period=${TARGET_PERIOD} \
-		--no-profile \
-		--no-gcp-trace-enabled
```

Comment:
`disable-expired-trial-projects` and `delete-user-data` are not used in this repository.